docs/node/workflow-scopes-and-cancellation.md

```diff
@@ -0,0 +1,121 @@
+# Scopes and Cancellation
+
+Temporal workflows have different types that can be cancelled:
```

Comment:
Workflows, Activities, Workers, Task Queues, Signals, Queries should all be capitalized in documentation, code comments, and readable output.